vmware/ngc/NGC-AdapterManager/src/main/java/org/opensds/vmware/ngc/adapters/opensds/RestClient.java

```diff
@@ -275,4 +278,63 @@ JSONObject detachVolume(String volumeId) throws Exception {
         }
         return responseDelete;
     }
+
+    JSONObject getVolume(String volumeId) throws Exception {
+
+        JSONArray response = (JSONArray)request.get(String.format("/block/volumes?wwn=%s", volumeId));
```

Comment:
Based on the Unique identifier implemntaion in hotpot , we have to chage this filter (?wwn) . 
issue raised https://github.com/opensds/nbp/issues/286